frontend/pages/software/ManageSoftwarePage/ManageSoftwarePage.tsx

```diff
@@ -209,7 +209,7 @@ const ManageSoftwarePage = ({
 
   useEffect(() => {
     setFilterVuln(location?.query?.vulnerable === "true" || false);
-    setPage(location?.query?.page || 0);
+    setPage(location?.query?.page ? parseInt(location?.query?.page, 10) : 0);
```

Comment:
You could end up with `NaN` here so you might want this to be `setPage(parseInt(location?.query?.page, 10) || 0);`

Reply:
@gillespi314 I was getting this error when I initially tried that:

<img width="836" alt="Screenshot 2023-04-27 at 9 42 14 AM" src="https://user-images.githubusercontent.com/71795832/234880723-795f2396-bf99-4ebd-b331-004fc0dd598b.png">

Thoughts?

Reply:
Maybe this?

```
const pageParam = queryParams.page || "";
setPage(parseInt(pageParam, 10) || 0);
```

Reply:
@gillespi314 realized I can just reuse the initialWhatever functions above

Reply:
Nice!